src/features/projects/components/ProjectSnippet.tsx

```diff
@@ -110,6 +110,7 @@ export default function ProjectSnippet({
       ) : null}
       <div
         onClick={() => {
+          setSelectedSite(0)
```

Comment:
There are some projects for which there are zero sites. The app crashes when such a project is selected. 

Relevant part of the error: 

```
Uncaught TypeError: Cannot read properties of undefined (reading 'type')
    at S (turf.min.js:1:5604)
    at Object.j [as bbox] (turf.min.js:1:9286)
    at zoomToProjectSite (zoomToProjectSite.ts:31:18)
```

e.g. eden-reforestation-projects on staging

Reply:
When the project snippet is visible, we do not always have access to the sites data as that is part of a different API call.